### DISCUSSION TITLE
Navmesh based Teleport ??

### DISCUSSION THREAD

**Discussion by davoodkharmanzar** [microsoft/MixedRealityToolkit-Unity#10150] [Q&A]
is it possible to teleport based on navmesh ??
against of surfaces and objects??

**Comment by RogPodge** [comment]
Hey there. Are you referring to having the user teleport to a fixed location, or something else. You can use the teleport hotspot to have the user move to fixed location when targetting a general area 

https://docs.microsoft.com/en-us/windows/mixed-reality/mrtk-unity/features/teleport-system/teleport-hotspot

Otherwise the teleport pointer should be able to target and let the user to teleport to any surface that is on the Valid Layers setting on the teleport pointer prefab:

![233424056_374613950736578_8289865667422901481_n](https://user-images.githubusercontent.com/39840334/129074949-0e45e85d-ee2b-479b-b545-3bf941be0469.png)
